src/vivarium_census_prl_synth_pop/constants/metadata.py

```diff
@@ -29,6 +29,13 @@
     "PUMA": "puma",
     "WGTP": "household_weight",
 }
+PERSONS_COLUMNS_TO_INITIALIZE = [
```

Comment:
Why are the new columns not in this list? Is it because this PR is _just_ loading it into the artifact and not into the state table?

Reply:
Correct.  Usually with how we do model versions is data artifact > implementation > observers/post-processing.

Reply:
Ah, makes sense!